### DISCUSSION TITLE
Conditional Dialogue Options

### DISCUSSION THREAD

**Discussion by ToniSewell** [nathanhoad/godot_dialogue_manager#577] [Q&A] [answered]
Hi,

I'm wondering if there's a way to have conditional dialogue options. Something as below:

```
~ Conversation

Floppy: Hello
- Me: Hi there!
if isHappy:
 - Me: I am happy today

=> END
```

Thank you !!

**Comment by Selenyhr** [comment, marked as answer]
Hi!

Yes, it is a feature, that is documented [here](https://github.com/nathanhoad/godot_dialogue_manager/blob/main/docs/Writing_Dialogue.md#conditions):
```
Nathan: What would you like?
- This one [if SomeGlobal.some_property == 0 or SomeGlobal.some_other_property == false]
    Nathan: Ah, so you want this one?
- Another one [if SomeGlobal.some_method()] => another_title
- Nothing => END
```

You just write your condition for a response within `[`brackets`]` :smile:

**Reply by ToniSewell** [reply]
thanks so much!
